### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by kesslerm**
Fixed various typos in BATCHING.md and the riak.conf template.

### CODE REVIEW

File: docs/BATCHING.md [hunk 1]
```diff
@@ -5,30 +5,30 @@ In Yokozuna versions prior to 2.0.4, update operations on Solr (notably, `add` a
 
 Yokozuna version 2.0.4 introduces batching and asynchronous delivery of Solr operations.  The primary objective of this work is to decouple update operations in Solr from the Riak vnodes that are responsible for managment of replicas of Riak objects across the cluster.  Without batching and asynchronous delivery, Riak vnodes have to wait for Solr operations to complete, sometimes an inordinate amount of time, which can have an impact on read and write operations in a Riak cluster, even for Riak objects that aren't being indexed through Yokozuna!  This feature is intended to free up Riak vnodes to do other work, thus allowing Riak vnodes to service requests from more clients concurrently, and increasing operational throughput throughout the cluster.
```
Comment (zeeshanlakhani) [inline docs/BATCHING.md:6]: This should be 2.0.7 as well, not 2.0.4.

**Comment by zeeshanlakhani** [comment]
Thank you @kesslerm. Mind updating the version at the top (2.0.4->2.0.7), rebasing into one typo's fix commit, and then we'll +1/merge?

**Comment by kesslerm** [comment]
Version number updated.

File: docs/BATCHING.md [hunk 2]
```diff
@@ -348,4 +348,4 @@ The `yz_solrq_drain_fsm` will stay in the waiting state until all of the tokens
 
 The relationship between the caller of the `drain` function (in this example, `yz_exchange_fsm`), the `yz_solrq_drain_fsm`, and the Solrq workers and helpers is illustrated in the following sequence diagram:
 
-![YS Solrq Draining](https://github.com/basho/internal_wiki/blob/master/images/yokozuna/yz-solrq-draining.png)
\ No newline at end of file
+![YS Solrq Draining](https://github.com/basho/internal_wiki/blob/master/images/yokozuna/yz-solrq-draining.png)
```
Comment (fadushin) [inline docs/BATCHING.md]: Oops.  Should be YZ, not YS

Comment (zeeshanlakhani) [inline docs/BATCHING.md]: nice catch.

Comment (fadushin) [inline docs/BATCHING.md]: Thanks for all the corrections, Magnus!

**Comment by kesslerm** [comment]
Re version number, is this the Riak version or the Yokozuna version? If the latter, it still builds as 2.0.3-*. Should Yokozuna jump some version numbers to 2.0.7, or was 2.0.4 the expected number?

**Comment by fadushin** [comment]
I originally meant it as the YZ version, which will got to 2.0.4, from 2.0.3.  I don't think we have any plans or precedent to align yz repo versions with riak versions, but I am fine if we say "pre-2.0.7" in the docs.  I doubt very much there are any other users of yokozuna than Riak.  :)

**Comment by zeeshanlakhani** [comment]
yeah @kesslerm @fadushin, maybe we say both (yz 2.0.4, riak version 2.0.7)

**Comment by kesslerm** [comment]
Let's stick with the yokozuna version here. Which YZ version will be in 2.1/2.2? Presumably the same?

**Comment by fadushin** [comment]
Yes, something similar, once this work is merged forward.

**Comment by kesslerm** [comment]
Removed the version number change patch again. From the way the document is written it is clear that it relates to the Yokozuna version, not the Riak version.

**Comment by kesslerm** [comment]
Pushed a new version of the patch that also fixes all occurences of YS instead of YZ. Please review again.

**Comment by fadushin** [comment]
+1 08b1185

**Comment by fadushin** [comment]
@borshop merge

**Pull request merged**